src/payment-account.js

```diff
@@ -13,6 +15,60 @@ class PaymentAccount extends ExchangePaymentAccount {
       return trade;
     });
   }
+
+  sell (bankId) {
+    return super.sell(bankId).then(trade => {
+      console.log('*** SELL TRADE CREATED ***', trade);
+      return trade;
+    });
+  }
+
+  add (obj) {
+    assert(obj, 'bank obj is required');
+
+    const b = {
+      account: {
+        currency: obj.account.currency,
+        bic: obj.account.bic,
+        number: obj.account.number
+      },
+      holder: {
+        name: obj.holder.name,
+        address: {
+          street: obj.holder.address.street,
+          city: obj.holder.address.city,
+          zipcode: obj.holder.address.zipcode,
+          country: obj.holder.address.country
+        }
+      },
+      bank: {
+        address: {
+          country: obj.bank.address.country,
+          street: obj.bank.address.street || null,
+          zipcode: obj.bank.address._ipcode || null,
+          city: obj.bank.address.city || null
+        }
+      }
+    };
+    return this._api.authPOST('bank-accounts', b).then(res => {
```

Comment:
Instead of deleting `bank.js`, it's better to move this stuff in there and create a class method, so you can do:  `return Bank.create(obj)`.